thrift/thrift_test.go

```diff
@@ -250,8 +257,9 @@ func TestRegisterPostResponseCB(t *testing.T) {
 		}
 
 		called := make(chan struct{})
-		cb := func(method string, response thrift.TStruct) {
+		cb := func(reqCtx context.Context, method string, response thrift.TStruct) {
 			assert.Equal(t, "Call", method)
+			assert.Equal(t, ctx.Value(key), value)
```

Comment:
nit: `Equal` should have expected argument before the value it's testing, so this should be,
```go
assert.Equal(t, value, ctx.Value(key))
```